validate.py

```diff
@@ -85,6 +86,34 @@ def get_reviews():
 def get_users():
   return list(sorted(os.listdir('players/')))
 
+def get_user_points():
+  points = {}
+
+  for user in get_users():
+    points[user] = 0
+
+    with open(os.path.join('players', user)) as inf:
+      try:
+        points[user] += int(inf.read())
+      except ValueError:
+        pass
+
+  cmd = ['git', 'log', 'master', '--first-parent', '--format=%s']
+  completed_process = subprocess.run(cmd, stdout=subprocess.PIPE)
+  if completed_process.returncode != 0:
+    raise Exception(completed_process)
+  process_output = completed_process.stdout.decode('utf-8')
+
+  merge_regexp = '^Merge pull request #[\\d]* from ([^/]*)/'
```

Comment:
This is a dangerous way of determining whether something is a merge commit or not. Someone could fill a PR with many commits, any one of which could contain a message that matches this that someone might not notice.

Almost makes me wonder if this is an _intentional_ oversight :)

Reply:
Learn something new every day!